src/screens/HoursScreen.tsx

```diff
@@ -1,17 +1,101 @@
 import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import { View, Text, StyleSheet, SafeAreaView, SectionList} from 'react-native';
 import baseStyle from '../styles/base';
+import colors, { theme } from '../styles/colors';
+import Constants from "expo-constants";
+
+const DATA = [
+  {
+    title: "Request A Walk",
+    data:
+    ['Sun                                 5:00pm-3:00am',
+     'Mon                                 5:00pm-1:00am',
+     'Tue                                 5:00pm-1:00am',
+     'Wed                                 5:00pm-1:00am',
+     'Thu                                 5:00pm-1:00am',
+     'Fri                                 5:00pm-1:00am',
+     'Sat                                 5:00pm-1:00am',
+
+    ]
+  },
+  {
+    title: 'Request SeeUs Van',
+    data: ['Sun                                 5:00pm-3:00am',
+     'Mon                                 5:00pm-1:00am',
+     'Tue                                 5:00pm-1:00am',
+     'Wed                                 5:00pm-1:00am',
+     'Thu                                 5:00pm-1:00am',
+     'Fri                                 5:00pm-1:00am',
+     'Sat                                 5:00pm-1:00am',]
+  },
+  {
+    title: 'Request A Jump/Unlock',
+    data: ['Sun                                 5:00pm-3:00am',
+     'Mon                                 5:00pm-1:00am',
+     'Tue                                 5:00pm-1:00am',
+     'Wed                                 5:00pm-1:00am',
+     'Thu                                 5:00pm-1:00am',
+     'Fri                                 5:00pm-1:00am',
+     'Sat                                 5:00pm-1:00am',]
+  }
+];
+
+const Item = ({title}) => (
```

Comment:
To fix the props validation error add a TypeScript type to this component's props like this:

```
const Item = ({ title }: { title: string }) => (
  <View style={styles.item}>
    <Text style={styles.title}>{title}</Text>
  </View>
);
```
Or can also be written as:

```
const Item = (props: { title: string }) => (
  <View style={styles.item}>
    <Text style={styles.title}>{props.title}</Text>
  </View>
);
```

That should get you past this error so you can commit the linted code.